docker-compose.yml

```diff
@@ -5,6 +5,6 @@ services:
     ports:
       - '5432:5432'
   rabbitmq:
-    image: 'rafakato/rabbitmq-delayed-message-exchange'
+    image: 'rabbitmq:3.6.10-alpine'
```

Comment:
Not sure if we need the `delayed-message-exchange` plugin yet. Will switch back if the need arises.